index.js

```diff
@@ -123,13 +111,25 @@ const baseOpen = async options => {
 
 		const browser = await defaultBrowser();
 		const browserName = browser.name.toLowerCase();
-		return baseOpen({
-			...options,
-			app: {
-				name: open.apps[browserName],
-				arguments: [...appArguments, flags[browserName]]
+
+		const supportedBrowsers = Object.keys(flags);
+		for (const supportedBrowser of supportedBrowsers) {
+			if (browserName.includes(supportedBrowser)) {
+				if (app === 'browserPrivate') {
+					appArguments.push(flags[supportedBrowser]);
+				}
+
+				return baseOpen({
+					...options,
+					app: {
+						name: open.apps[supportedBrowser],
+						arguments: appArguments
+					}
+				});
 			}
-		});
+		}
```

Comment:
I'm not too sure how to normalise the names, but is this good enough? It just checks if the `browserName` string contains `"chrome"`, `"firefox"` or `"edge"` since these are the only browsers supported right now.

Reply:
That is not going to work. What if the default browser is `Chrome Canary`?

Reply:
Sorry for the lack of replies. I don't know what to do besides converting to lowercase and stripping whitespace. Do you have any advice for normalisation?

Reply:
A list of browsers with their mapping on the different operating systems. We only support 3 browsers for this, so should not be hard to make a list like that.

For example, `.chrome` is `Google Chrome` on macOS, `google-chrome` on Linux.